presto-main/src/main/java/com/facebook/presto/operator/scalar/StringFunctions.java

```diff
@@ -56,7 +56,7 @@
 public final class StringFunctions
 {
     private StringFunctions() {}
-
+    
```

Comment:
There are a number of unrelated formatting changes in this commit. Please limit the changes to what's necessary to fix the issue.